spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -40,7 +40,16 @@ def handle_two_step(response)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
       elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Array) && r.body["trustedPhoneNumbers"].first.kind_of?(Hash)
-        handle_two_factor(r)
+        puts("Two Step Verification for account '#{self.user}' is enabled")
+        puts("Please select a phone number to verify your identity")
+
+        available = r.body["trustedPhoneNumbers"].collect do |current|
+          "#{current['numberWithDialCode']}\tSMS\t(#{current['id']})"
```

Comment:
pushmode seems to be SMS all the time currently, but let's make this more flexible:
```suggestion
          "#{current['numberWithDialCode']}\#{current['pushMode]}\t(#{current['id']})"
```
(Source: https://github.com/fastlane/fastlane/pull/13554/files#diff-4d6113a73b52b07c6bee2f1f2b52ce40R129)